zebra-network/src/constants.rs

```diff
@@ -116,7 +116,7 @@ pub const USER_AGENT: &str = "/Zebra:1.0.0-alpha.17/";
 ///
 /// The current protocol version typically changes before Mainnet and Testnet
 /// network upgrades.
-pub const CURRENT_NETWORK_PROTOCOL_VERSION: Version = Version(170_013);
+pub const CURRENT_NETWORK_PROTOCOL_VERSION: Version = Version(170_014);
```

Comment:
I double-checked that this matches the testnet network protocol version in https://github.com/zcash/zips/blob/main/zip-0252.rst#nu5-deployment